packages/client/hmi-client/src/utils/bitmask.ts

```diff
@@ -0,0 +1,76 @@
+enum OperatorInteractionStatus {
+	Found = 0x0001,
+	Hover = 0x0010,
+	Focus = 0x0100,
+	Drag = 0x1000
+}
```

Comment:
Because it is not exported, feel free to keep the name simple to make your code less verbose if you want.
```suggestion
enum Status {
	Found = 0x0001,
	Hover = 0x0010,
	Focus = 0x0100,
	Drag = 0x1000
}
```

Reply:
Makes sense, I just put a comment to clarify that its for interactions